docs/viz.md

```diff
@@ -29,7 +29,7 @@ Parameter | Description | Example
 data | GeoJSON Feature Collection or JSON Join-Data | 'points.geojson'
 vector_url | optional property to define vector data source (supported for basic MapViz, CircleViz, GraduatedCircleViz, HeatmapViz, ChoroplethViz, LinestringViz) | 'mapbox://mapbox.mapbox-terrain-v2'
 vector_layer_name | property to define target layer of vector source | 'contour'
-vector_join_property | property to aid in determining color for styling vector layer | 'ele'
+vector_join_property | property of features in vector tile data to use as link to joined json data | 'ele'
```

Comment:
`Name of the property in the reference vector tile source to join to the `JSON` data passed to this function